gltf/src/net/mgsx/gltf/loaders/shared/geometry/MeshLoader.java

```diff
@@ -236,12 +237,20 @@ else if(attributeName.startsWith("_")){
 									normalMapUVs = attribute;
 								}
 							}
-							if(normalMapUVs == null) throw new GLTFIllegalException("UVs not found for normal map");
+							if(normalMapUVs == null) {
+								// fallback to default normal attribute and avoid npe
+								normalMapUVs = VertexAttribute.Normal();
+							}
 						}
 					}
 				}
+
+				// clamp vertex to maximum possible
+				final int maxVertexAttrs = Math.min(vertexAttributes.size, PBRCommon.getCapability(GL20.GL_MAX_VERTEX_ATTRIBS));
```

Comment:
again, it would silently fail and render broken models, which is not the solution here. I also remove random attributes which could be a problem (eg. position is not necessary the first attribute).

I guess this is due to too many shape keys (morph targets) so a cleaner solution would be to truncate the array by removing the morph target with less importance (that is what the spec says, iirc) and at least log a warning.

Finding the morph target with less importance could be a bit tricky i guess because of animations but i think you could use current node weights values.